.gitignore

```diff
@@ -16,6 +16,7 @@
 *.launch
 .settings/
 *.sublime-workspace
+.yarn
```

Comment:
```suggestion
```

Reply:
If you're moving this repo over to Yarn 3, please see the Yarn docs for what needs to be in the .gitignore.